website/docs/developer/deployment-guide/deployment-upgrading-orca.md

```diff
@@ -0,0 +1,102 @@
+---
+id: deployment-upgrading-orca
+title: Upgrading ORCA
+description: Provides developer with information on upgrading ORCA.
+---
+
+After the initial ORCA deployment, any future updates to the ORCA deployment 
+from configuration files, Terraform files (`*.tf`), or modules from a new 
+version of ORCA can be deployed and will update the appropriate portions of 
+the stack as needed.
+
+**IMPORTANT:**  For instances where ORCA has been deployed with Cumulus, 
+visit the [Supported PI Versions wiki page](https://wiki.earthdata.nasa.gov/display/CUMULUS/Supported+PI+Versions) 
+to ensure version compatibility.
+
+## ORCA Versioning
+
+ORCA uses a global versioning approach, meaning version numbers are 
+consistent across all Terraform modules and semantic versioning to track 
+major, minor, and patch version (e.g., 1.0.0).
+
+**IMPORTANT:** By convention, ORCA minor version releases introduce breaking 

```

Comment:
Change this **IMPORTANT** block to an admonition.

Change `ORCA minor version releases introduce breaking changes (e.g., 1.13.x -> 1.14.0), so it is critical that the release notes are viewed for migration steps`
to something like `ORCA major version releases (e.g., 2.x.x -> 3.x.x) introduce known breaking changes. However, any version change has the possibility to introduce breaking changes for your particular use case. It is critical that  the release notes are viewed for migration steps and changes.`

Reply:
made update to add the admonition